moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -743,6 +741,47 @@ bool JogCalcs::updateJoints(JogArmShared& shared_variables)
   // Cache the original joints in case they need to be reset
   original_joint_state_ = internal_joint_state_;
 
+  // Calculate worst case joint stop time, for collision checking
+  std::string joint_name = "";
+  shared_variables.worst_case_stop_time = std::numeric_limits<double>::max();
+  moveit::core::JointModel::Bounds kinematic_bounds;
+  double accel_limit = 0;
+  double joint_velocity = 0;
+  double worst_case_stop_time = 0;
+  for (size_t jt_state_idx = 0; jt_state_idx < incoming_joint_state_.velocity.size(); ++jt_state_idx)
+  {
+    joint_name = incoming_joint_state_.name[jt_state_idx];
+
+    // Get acceleration limit for this joint
+    for (auto joint_model : joint_model_group_->getActiveJointModels())
+    {
+      if (joint_model->getName() == joint_name)
+      {
+        // Some joints do not have acceleration limits
+        try
+        {
+          kinematic_bounds = joint_model->getVariableBounds();
+          // Be conservative when calculating overall acceleration limit from min and max limits
+          accel_limit =
+              std::min(fabs(kinematic_bounds[0].min_acceleration_), fabs(kinematic_bounds[0].max_acceleration_));
```

Comment:
I think we can safely take `min_acceleration_` here as it's defined and implemented as min/max range (with min value being the negative max value by default).

Reply:
Basically, assume they are symmetric? That's OK with me